src/file_format.rst

```diff
@@ -0,0 +1,189 @@
+================================
+Introduction to the file formats
+================================
+
+LAMMPS internally keeps track of the evolution of atomistic systems. Such a
+system is described as a `simulation box` containing an ensemble of particles
+interacting with one another according to set of rules, and each with their own
+position, velocities etc.
+
+In this section, you will learn how to create a very simple simulation box
+using LAMMPS built-in tools.
+
+Running a minimal LAMMPS input
+------------------------------
+
+In an empty directory, open a text file and copy the following text inside:
+
+.. code-block:: LAMMPS
+
```

Comment:
```suggestion
atom_style atomic
```

My preference is, since we introduce `atom_style` a bit later, we should make this line explicit. Alternatively we could delay discussing `atom_style` at all until (for example) we get to a molecular simulation.